app/src/main/java/org/mozilla/fenix/addons/AddonsActivity.kt

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.addons
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import org.mozilla.fenix.R
+
+/**
+ * An activity to manage add-ons.
+ */
+class AddonsActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_addons)
+
+        if (savedInstanceState == null) {
+            supportFragmentManager.beginTransaction().apply {
+                replace(R.id.container, AddonsFragment())
```

Comment:
nit: For navigation Fenix is using [the Navigation library](https://www.youtube.com/watch?v=JFGq0asqSuA). As above let's land as it is and do a follow up.

Reply:
https://github.com/mozilla-mobile/android-components/issues/5374